src/styles/stars.css

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
```

Comment:
how do you imagine it?
use integer values

Reply:
and this one